files/templates/alertmanager/notification-template.tmpl

```diff
@@ -1,13 +1,23 @@
 {{ define "__alertmanager" }}Alertmanager{{ end }}
-{{ define "__alertmanagerurl" }}{{ .ExternalURL }}/#/alerts?receiver={{ .Receiver }}&silenced=false&inhibited=false&active=true&filter=%7Balertname%3D%22{{ .CommonLabels.alertname }}%22%7D{{ end }}
+
+{{ define "__alertmanagerurl" }}
```

Comment:
I think we can not define the alertmanager and alertmanager url values if mimir is enabled right?

Reply:
The alertmanager url is used as a link in the title.
To avoid more refactoring I kept the variable but changed the value in case of mimir.
But I can refactor  properly if you prefer :-)

Reply:
Well, we should remove it from the slack title link as well :D

Reply:
Why ?
We can have a direct link to grafana on the title, no ?

Reply:
Why would we link to alertmanager as we removed the ingress?

Reply:
Oh yes that's because you replaced the alertmanagerurl value sure